mindmeld/components/entity_resolver.py

```diff
@@ -593,7 +377,7 @@ def fit(self, clean=False):
             return
 
         # load data: list of canonical entities and their synonyms
-        entity_map = self._load_entity_map()
+        entity_map = self._format_entity_map(entity_map) or self._load_entity_map()
```

Comment:
looks like we would only format entities when they come from input arg but not when they are loaded from file?

Reply:
modified, we are now formatting both during loading as well as when it is inputted